src/swap/defi/lifi.ts

```diff
@@ -125,7 +127,7 @@ const asExchangeInfo = asObject({
 const asEstimate = asObject({
   fromAmount: asNumberString, // "400000",
   toAmount: asNumberString, // "237318132569913",
-  toAmountMin: asNumberString, // "225452225941418",
+  // toAmountMin: asNumberString, // "225452225941418",
```

Comment:
unused cleaner params should be commented, not removed. Same as for `expectedOutputMaxSlippage` we don't show the user the 'worse case' quote anymore since it's a variable quote